### PR TITLE
[1] Added script to make it easier to update secrets

### PR DESCRIPTION
**Pull request by Xartos**
**What this PR does / why we need it**:

I had this scrip laying around so I thought I would add it here.

Makes it easier to generate secrets for a new deployment (Or to update existing ones)

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Review by OlleLarsson — APPROVED** [review]

**Comment by OlleLarsson** [comment]
Maybe we could name the script `generate-passwords`, "generate-secrets", or similar?
I kind of got the association with `update-passwords` that it would update passwords that are already deloyed :)

File: scripts/update-passwords.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  yq w - 'elasticsearch.snapshotterPassword' "$(pwgen -cns 20 1)" | \
+  yq w - 'elasticsearch.metricsExporterPassword' "$(pwgen -cns 20 1)" | \
+  yq w - 'elasticsearch.kibanaCookieEncKey' "$(pwgen -cns 32 1)" | \
+  yq w - 'kubeapiMetricsPassword' "$(pwgen -cns 20 1)" | \
+  yq w - 'dex.staticPasswordNotHashed' "${DEX_STATIC_PASS}" | \
+  yq w - 'dex.staticPassword' "${DEX_STATIC_PASS_HASH}" | \
+  yq w - 'dex.kubeloginClientSecret' "$(pwgen -cns 20 1)" | \
+  yq w - 'user.grafanaPassword' "$(pwgen -cns 20 1)" | \
+  yq w - 'user.alertmanagerPassword' "$(pwgen -cns 20 1)" | \
+  yq w - 'prometheus.remoteWrite.password' "$(pwgen -cns 20 1)" | \
```
Comment (viktor-f) [inline scripts/update-passwords.sh:54]: This must be the same as `influxDB.users.wcWriterPassword`
```suggestion
  yq w - 'prometheus.remoteWrite.password' "$(<read from influxDB.users.wcWriterPassword>)" | \
```

**Review by viktor-f — CHANGES_REQUESTED** [review]
Very nice!

I would also like to rename it like Olle suggested.
I also suggest that you mention it in the readme and the public docs.

File: scripts/update-passwords.sh (hunk 1 above)
Comment (OlleLarsson) [reply scripts/update-passwords.sh:54]: Good catch.

**Review by viktor-f — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]

**Pull request merged**